mcpo/entrypoint.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+echo "Running uname -m..."
+ARCH="$(uname -m)"
+echo "Raw architecture output: $ARCH"
+
+case "$ARCH" in
+  x86_64) ARCH_TAG="amd64" ;;
+  aarch64|arm64) ARCH_TAG="arm64" ;;
+  *)
+    echo "Unsupported architecture: '$ARCH'"
+    exit 1
+    ;;
+esac
+
+echo "Detected architecture tag: $ARCH_TAG"
+
+if [ -e "/var/run/docker.sock" ]; then
```

Comment:
You could also check if the docker CLI is already installed. That way you don't have to re-download it every time Rancher Desktop is restarted.

Reply:
Added a check for all the tools and downloads them only if they are not present already.